rpc/server_options.go

```diff
@@ -33,6 +33,12 @@ type serverOptions struct {
 	// allowUnauthenticatedHealthCheck allows the server to have an unauthenticated healthcheck endpoint
 	allowUnauthenticatedHealthCheck bool
 
+	// allowPublicMethods allows for nested public API methods on the server
+	// i.e. will disable auth checks for users
+	allowPublicMethods bool
```

Comment:
This is never checked anywhere

Reply:
sorry yeah, something i need to remove. I had it in there originally but the map is enough.

Reply:
hm weird actually, my commit didn't push up with this removed